csympy/lib/csympy_wrapper.pyx.in

```diff
@@ -6,6 +6,12 @@ from libcpp.string cimport string
 from cpython cimport PyObject, Py_XINCREF, Py_XDECREF, \
     PyObject_CallMethodObjArgs
 
+DEF HAVE_SAGE = ${HAVE_SAGE}
+have_sage = ${HAVE_SAGE}
```

Comment:
The disadvantage of this is that the Python wrapper depends on how csympy was configured. You also lose syntax highlighting in e.g. Vim. I wonder if you can just try to import `sage.rings.integer`, and if it succeeds, define HAVE_SAGE as True, otherwise False.